src/api/app/views/webui/users/bs_requests/index.html.haml

```diff
@@ -0,0 +1,65 @@
+- @pagetitle = requests_listing_pagetitle
```

Comment:
We don't need to duplicate the whole content in this file. We already have it under https://github.com/openSUSE/open-build-service/blob/master/src/api/app/views/webui/shared/bs_requests/_form.html.haml.

We can just render this partial here:


```suggestion
- @pagetitle = requests_listing_pagetitle(@project)

.row
  .px-0.px-md-3.mb-3
    .card
      = render(partial: 'webui/package/tabs', locals: { project: @project, package: @package })

= render(partial: 'webui/shared/bs_requests/form',
         locals: { selected_filter: @selected_filter, bs_requests_creators: @bs_requests_creators, url: @url, bs_requests: @bs_requests })
```